cmd/influx/main.go

```diff
@@ -425,12 +425,9 @@ func (c *CommandLine) parseInto(stmt string) string {
 }
 
 func (c *CommandLine) Insert(stmt string) error {
-	stmt = stmt[7:]
-	if strings.EqualFold(stmt[:4], "into") {
-		stmt = c.parseInto(stmt[5:])
-	}
-	if c.RetentionPolicy == "" {
-		c.RetentionPolicy = "default"
+	stmt = strings.TrimSpace(strings.TrimSpace(stmt)[6:])
```

Comment:
Even something as simple as `INSERT_TOKEN_POS` would be better. It might be overkill, but `6` is pretty random looking.